src/schemas/CreateBrand.json

```diff
@@ -0,0 +1,14 @@
+{
+    "title": "createBrand",
+    "type": "array",
```

Comment:
We will modify this structure for something like this:
```
{
"brands": [
     {...}
  ]
}
```
This is necessary because we will then generate the types of this endpoint for use in the FE, and the actual structure will result in an array of a generic object.